client/src/analysis/Functions.res

```diff
@@ -63,7 +63,8 @@ let calculateUnsafeUserFunctions = (props: props, t: t): Set.String.t => {
       if uf.name != "" {
         E.filterMap(FluidAST.toExpr(uf.body), ~f=x =>
           switch x {
-          | EBinOp(_, callee, _, _, _) => Some(PT.InfixStdlibFnName.toString(callee), uf.name)
+          | EInfix(_, InfixFnCall(callee, _), _, _) =>
+            Some(PT.InfixStdlibFnName.toString(callee), uf.name)
```

Comment:
Because we're only looking at user functions, and BinOps are not user functions.

(In practice, user functions can't be binops anyway, so this is overkill, but theoretically they could be in the future)